bsmetadata/metadata_processors.py

```diff
@@ -128,10 +128,18 @@ def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
         # Example: "foo.bar/Year 2021/" instead of "foo.bar/Year%202021/".
         return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, unquote_plus(metadata_attrs["value"])])
 
+class WebsiteDescriptionProcessor(MetadataProcessor):
+    """An example metadata processor for website descriptions."""
+
+    def process_global(self, metadata_attrs: Dict[str, Any]) -> Optional[str]:
+        # Example: "website_description: BBC is a news organization".
+        return "".join([metadata_attrs["key"], self.cfg.metadata_key_value_sep, metadata_attrs["value"]])
```

Comment:
I think I'd personally favour a variant where "Website Description" is written in plain English (instead of "website_description") - that is, ` return "".join("Website Description", self.cfg.metadata_key_value_sep, metadata_attrs["value"]])` - but I guess that's just a matter of taste :)

Reply:
Now that you mention it, I too prefer "Website Description" :) Added the change and modified tests accordingly!